feat: alias -s to --show, update docs (#105)
charmbracelet/mods: README.md

go install github.com/charmbracelet/mods@latest
```

## Saving conversations

All conversations are automatically saved, and can be identified by their latest
prompt.

A saved conversation has a SHA1 identifier and a title, similar to how Git
works.

> Not so similar to Git, conversations can be updated, keeping their id but having

Reviewer: I would change this to "Unlike Git, conversations [...]"
```suggestion
Unlike Git, conversations can be updated, keeping their id but having
```

We’d love to hear your thoughts on this project. Feel free to drop us a note.

- [Twitter](https://twitter.com/charmcli)
- [The Fediverse](https://mastodon.social/@charmcli)
- [Discord](https://charm.sh/chat)

## License

[MIT](https://github.com/charmbracelet/mods/raw/main/LICENSE)

---

Part of [Charm](https://charm.sh).

<a href="https://charm.sh/"><img alt="The Charm logo" width="400" src="https://stuff.charm.sh/charm-badge.jpg" /></a>

> <!--prettier-ignore-->

Reviewer: What does this do?

Author: makes prettier not f up the line bellow :P
